src/backend/src/plugins/telemetry/telemetry_service_impl.h

```diff
@@ -1115,7 +1115,7 @@ class TelemetryServiceImpl final : public rpc::telemetry::TelemetryService::Serv
 
         _telemetry.home_async(
             [this, &writer, &stream_closed_promise, is_finished, &subscribe_mutex](
-                const mavsdk::Telemetry::Position& home) {
+                const mavsdk::Telemetry::Position home) {
```

Comment:
Do we want to remove that `&`, or is that a mistake in the template?

Reply:
Hm, this probably happened by mistake. In general I would move to pass by value / copy as much as possible and then profile and optimize if needed.

Reply:
Ok